docs/preprocessing.md

```diff
@@ -0,0 +1,15 @@
+## Executing *fMRIPrep*
+Because *fMRIPrep* creates a single anatomical reference for all sessions, we will need first to run with ``--anat-only`` so that the results get cached and *fMRIPrep* processes do not run into race conditions.
+
+Similarly, when running *fMRIPrep* on a dataset with many sessions, the safest approach is to submit one job per session.
```

Comment:
```suggestion
Because *fMRIPrep* creates a single anatomical reference for all sessions, we generate such reference first by setting the ``--anat-only`` flag.
If that *fMRIPrep* execution finishes successfully, the anatomical processing outcomes will be stored in the output folder.
We will then run one *fMRIPrep* process for each dataset's session, which is the recommended way for datasets with a large number of sessions (e.g., more than six sessions).
We avert that session-wise *fMRIPrep*'s processes run into race conditions by pre-computing the anatomical reference.
```